utils/audioprocessing/processing.py

```diff
@@ -589,3 +589,19 @@ def alarm_handler(signum, frame):
     signal.alarm(0)
     if process.returncode != 0 or not os.path.exists(output_filename):
         raise AudioProcessingException(stdout)
+
+
+def analyze_using_essentia(essentia_executable_path, input_filename, output_filename_base, essentia_profile_path=None):
+    """
+    runs Essentia's FreesoundExtractor analsyis
+    """
+    essentia_dir = os.path.dirname(os.path.abspath(essentia_executable_path))
+    os.chdir(essentia_dir)
```

Comment:
I'm not sure this is necessary - why can't you run it from any directory? as long as the binary path and profile path are absolute paths (which I thinkthey should be)

Reply:
addressed in a4d1d259d28a6f0355a913472059970101cfc9df